index.html

```diff
@@ -1,11 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8">
     <link rel='icon' href="/favicon.ico">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Travel Companion</title>
-  </head>
+
+   
+    <!--Tracys implmentation changes for Index.html-->
+    <!--This is to link Semantic UI for user location design-->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.5.0/dist/semantic.min.css">
```

Comment:
It looks like you are using a new CSS library here. We already have Bootstrap for CSS, is there any specific reason for this inclusion? It doesn't matter which one we end up using, but we should only use 1 to keep a consistent look throughout the site.

Reply:
I dont fully remeeber at this time, but I can review the code, I want to say this was only added for the sake of a singular button... if so I can go through and try to use the bootstrap library, thatll be something i got to dig into